linera-service/Cargo.toml

```diff
@@ -54,7 +54,7 @@ kubernetes = [
     "dep:pathdiff",
     "dep:fs_extra",
 ]
-existing-net = []
+net-with-faucet = []
```

Comment:
`faucet = []`

Reply:
Wouldn't `faucet-net` be better here? Since this isn't general `faucet` functionality, but specific to the faucet net 🤔 I'll do `faucet-net`, but I can switch to `faucet` if you disagree

Reply:
None of that matters because we want to change things in depth

Reply:
Ok, just left it as `faucet-net` for now then!